pkg/reconciler/instances/istio/action.go

```diff
@@ -71,6 +71,29 @@ func (a *MainReconcileAction) Run(context *service.ActionContext) error {
 		return err
 	}
 
+	err = deployIstio(context, performer)
+
+	errPatchMutatingWebhook := performer.PatchMutatingWebhook(context.Context, context.KubeClient, context.Logger)
+	if errPatchMutatingWebhook != nil {
+		errPatchMutatingWebhook = errors.Wrap(errPatchMutatingWebhook, "Could not patch MutatingWebhookConfiguration")
+	}
+
+	if err != nil {
+		if errPatchMutatingWebhook != nil {
+			return errors.Wrap(err, errPatchMutatingWebhook.Error())
+		}
+
+		return err
+	}
+
+	if errPatchMutatingWebhook != nil {
```

Comment:
Do we really need another error variable?

Reply:
yes, we have 4 different cases (2 per method)